development/__init__.py

```diff
@@ -1 +1,5 @@
-ACTIONS = '''<a href="{% url 'development:match_detail' record.pk %}">view</a>'''
+from environment import get_env_variable
+
+
+SERVER_URL = get_env_variable('SERVER_URL')
+SERVER_PORT = get_env_variable('SERVER_PORT')
```

Comment:
Could you move these vars to Django settings?

https://github.com/evbeda/edagames-django/blob/main/edagames/settings.py

Reply:
done!